src/westpa/core/binning/mab.py

```diff
@@ -243,6 +246,18 @@ def map_mab(coords, mask, output, *args, **kwargs):
         # output is the main list that, for each segment, holds the bin assignment
         output[i] = holder
 
+    if bin_log and report:
+        if westpa.rc.sim_manager.n_iter:
+            with open(expandvars("$WEST_SIM_ROOT/binbounds.log"), 'a') as bb_file:
```

Comment:
Also, I wonder if this path is something that should be configurable in `west.cfg`.

Reply:
This was intended to be a "first pass" (and thus minimally coupled to west.cfg and the data manager) since we're trying to get 2022.06 out to fix our pypi install. 

Definitely should be user-customizable in the future!